clients/src/main/java/org/apache/kafka/common/protocol/ApiKeys.java

```diff
@@ -123,6 +124,9 @@
     /** indicates whether the API is enabled for forwarding **/
     public final boolean forwardable;
 
+    /** indicates if this API is exclusive to the KIP-500 mode **/
+    public final boolean isKip500OnlyApi;
```

Comment:
@hachikuji I am not entirely sure we need this additional flag but I don't think there's a cleaner way to mark certain APIs as being available only under KIP-500